packages/astro/src/vite-plugin-astro-server/trailing-slash.ts

```diff
@@ -22,6 +22,9 @@ export function trailingSlashMiddleware(settings: AstroSettings): vite.Connect.N
 			/* malformed uri */
 			return next(e);
 		}
+		if(pathname.startsWith('/_') || pathname.startsWith('/@')) {
```

Comment:
I would use `@vite` to avoid a possible clash with other tools or user-defined endpoints.

Reply:
There's also `@id` and `@fs` and possibly others though. The failure mode is to just not show the helpful error page, so I'd rather err on the side of skipping too many

Reply:
😭